scripts/rnnlm/train_rnnlm.sh

```diff
@@ -23,6 +23,8 @@ final_effective_lrate=0.0001
 embedding_l2=0.005
 embedding_lrate_factor=0.1  # the embedding learning rate is the
                             # nnet learning rate times this factor.
+backstitch_scale=0.0        # backstitch training scale
```

Comment:
similar to the comment above? does 0.0 mean no backstitch?

Reply:
Yes.